nominees/open-insurance-management-information-system.json

```diff
@@ -29,14 +33,9 @@
   "repositoryURL": "https://github.com/openimis/web_app_vb",
   "organizations": [
     {
-      "name": "GIZ",
+      "name": "Deutsche Gesellschaft für Internationale Zusammenarbeit (GIZ)",
       "website": "https://www.giz.de/en/html/index.html",
       "org_type": "owner"
-    },
-    {
-      "name": "OpenIMIS",
```

Comment:
@nathanbaleeta plz leave this one in place. Thanks

Note that the submission form only allows to include one organization, so whenever there is more than one; keep them unless there is a good reason to remove them.

Reply:
@lacabra have reincluded `OpenIMIS` as one of the organizations. Thanks.